src/components/AddCard/_add-card.scss

```diff
@@ -15,4 +15,8 @@
   svg {
     margin: auto 0 0 auto;
   }
+
+  p.addcard-title {
```

Comment:
I would like us to not add to the specificity by using the element selectors.

Reply:
Removed specificity